src/test/resources/iudx-file-server-api.Release-v4.0.postman_collection.json

```diff
@@ -49,14 +49,13 @@
 							}
 						},
 						"url": {
-							"raw": "https://{{auth-url}}/auth/v1/token",
+							"raw": "https://{{auth-url}}/{{dxApiBasePath}}/token",
```

Comment:
Use Correct Auth path, it should be `{{dxAuthBasePath}}`

Reply:
Thank you for identifying that, I'll correct it right away.

Reply:
Corrected it [here](https://github.com/datakaveri/iudx-file-server/pull/125/commits/c7f52bf76a736782d15de01a12a1326c3d0437d9)